### ISSUE TITLE
Document and publish our spec FHIR IG for lab orders we are using for ETOR

### ISSUE THREAD

**Issue by MauriceReeves-usds** [CDCgov/prime-reportstream#10382]
Because there is no proposed IG for a lab order yet in FHIR, we should find a place to document what format we're currently going to share with TI, and we should also look to present the format to the working group at HL7 that is making the draft IG now. It would be good to get this done soon so they have an opportunity to provide feedback or recommendations.

cc: @avnieldravid @jimmyfagan @johnnking

**Comment by avnieldravid** [comment]
As discussed in the ETOR meeting, this work should be ticketed on the TI board.  The RS team doesn't feel equipped to do this work. Iceboxing for now.

**Comment by JohnNKing** [comment]
@avnieldravid Yes, though I'd like to defer it for a couple weeks due to folks being out of the office.
